### ISSUE TITLE
Encrypting the backup file with a password

### ISSUE THREAD

**Issue by serrq** [quillpad/quillpad#136]
I would like to encrypt the backup file within the app.

**Comment by githubcatw** [comment]
I agree, and want to suggest being able to encrypt individual notes, like in Samsung Notes for example. I can implement this but I don't know if I should start now or wait until 1.5 and the local storage rewrite.
